src/test/java/org/circle8/integration/recorrido/RecorridoPostFinTest.java

```diff
@@ -0,0 +1,37 @@
+package org.circle8.integration.recorrido;
+
+import java.sql.Timestamp;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+
+import io.restassured.RestAssured;
+
+import org.circle8.ApiTestExtension;
+import org.circle8.utils.Dates;
+import org.exparity.hamcrest.date.DateMatchers;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import lombok.val;
+
+@ExtendWith(ApiTestExtension.class)
+public class RecorridoPostFinTest {
+	@Test
+	void testPostOk() {
+		val time = ZonedDateTime.now(Dates.UTC).toInstant();
+		val response =RestAssured.given()
+					  .post("/recorrido/1/fin")
+					  .then()
+					  .statusCode(200);
```

Comment:
estaria bueno chequear que en la DB realmente se haya cambiado la fecha de fin

Reply:
Pero no veo la forma de verificarlo. Si genero una fecha para contrastar siempre va a tener un margen de error con lo que efectivamente se grabe, y ese margen de error puede hacer que se caiga el test.

Reply:
en la DB haces un `SELECT` para chequear que el `Recorrido` con `ID = 1` tenga una `FechaFin IS NOT NULL`

Reply:
Para simplificar pregunté en el body si incluye el campo.